Source/Resources/Content/MSGraph/Groups/BUILTIN - Device Administrators.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "This group is used to assign users that should be added as local administrators on Azure AD joined devices.",
+  "description": "What is it: A manual group used to assign local administrators to Azure AD joined devices. Members in this group are granted the Azure AD Role 'Device Administrators'. \n Simeon Notes: This should be a restricted group with only system administrators included.",
```

Comment:
**What does this do?**
Creates a group of users assigned as local administrators on Azure AD joined devices. The baseline grants users in the group the Azure AD Role 'Device Administrators." This should be a restricted group with only system administrators included.

**Why should you use this?**
If you want to have a group of users with local administrator permissions on Azure AD joined devices.

**What is the end user impact?**
A certain group of users will become local administrators for Azure AD joined devices. 

**Learn more**
NA